### DISCUSSION TITLE
What are the conditions to (not) show the right click menu option?

### DISCUSSION THREAD

**Discussion by PeterGeelen** [mjisaak/skilling-champion-extension#12] [Q&A]
Hello Martin,

just to understand how the MVP D&L Champion extension works, is there a reason that the extension doesn't show in the right click menu when you click on certain web links?

Eg. https://docs.microsoft.com/en-us/answers/products/
All menu links on top left show the extension option, except for the FAQ & Help.

Could it be that the link must point to the same website as the source page.
FAQ & Help points at https://aka.ms/msftqahelp.
While the other menu items refer to pages on docs...

Is this correct?
Because that hides/disables the extension in many cases...

**Comment by brunolewin** [comment]
Hi Peter, nice to see you here. I'll take that old Q since I played with the code a bit...

The extension has a list of sites that support Creator IDs and will only show for links to these sites. You can see the list in [src/main.js](https://github.com/mjisaak/mvp-docs-learn-champion-extension/blob/master/src/main.js).

**Reply by PeterGeelen** [reply]
OK, thx for the info!

Is that list defined by technical requirements of the webserver? 
Or can we add suggestions? (like Wiki ;) )

Rgds,
Peter
